src/app/[locale]/viewer/_components/submodel-elements/generic-elements/entity-components/EntityComponent.tsx

```diff
@@ -3,7 +3,7 @@ import { ChevronRight, ExpandMore } from '@mui/icons-material';
 import { SubmodelViewObject } from 'lib/types/SubmodelViewObject';
 import { generateSubmodelViewObjectFromSubmodelElement } from 'lib/util/SubmodelViewObjectUtil';
 import { SimpleTreeView } from '@mui/x-tree-view';
-import { EntityTreeItem } from './EntityTreeItem';
+import { EntityTreeItem } from 'app/[locale]/viewer/_components/submodel-elements/generic-elements/entity-components/EntityTreeItem';
```

Comment:
Interestingly all new imports changed to absolute paths. Ok for me - My IDE does the same but in the past relative paths were used.

Reply:
I would prefer relative paths as well.